### PR TITLE
Change roles to clusterroles for operators

### PR DESCRIPTION
**Pull request by DomFleischmann**
This will change the katib-controller and katib-uiroles to clusterroles.

Additionally Dominik Fleischmann is being added to the owners of the katib operators.

### CODE REVIEW

**Comment by DomFleischmann** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> memo **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> #### What to do if you already signed the CLA
> ##### Individual signers
> 
>     * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> 
> ##### Corporate signers
> 
>     * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> 
>     * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
>     * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> 
> information_source **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fkubeflow%2Fkatib%2Fpull%2F1426) for more info**.

I've signed the CLA so if the check is re triggered it should now pass.

File: operators/OWNERS [hunk 1]
```diff
@@ -1,3 +1,4 @@
 approvers:
   - knkski
   - rfmvasconcelos
+  - DomFleischmann
```
Comment (andreyvelich) [inline operators/OWNERS:4]: Can you make it in alphabetical order please?

**Review by andreyvelich — COMMENTED** [review]
Thank you for the contribution @DomFleischmann!
/ok-to-test

**Comment by RFMVasconcelos** [comment]
@DomFleischmann this looks good to me. 

cc @andreyvelich, @DomFleischmann has recently joined the team as another colleague for Kubeflow ventures. As he has plenty of operator experience, I think it's great to have him as an approver here.

I'm happy to remove myself if you think we should stick to 2 people as @knkski and @DomFleischmann work with this code more on a daily basis. 

/lgtm 
/approve

File: operators/OWNERS (hunk 1 above)
Comment (RFMVasconcelos) [reply operators/OWNERS:4]: Damn, I was about to merge this! Nice catch.

Comment (RFMVasconcelos) [reply operators/OWNERS:4]: Should we remove myself to stick to 2 people? As I'm more on the docs these days?

**Comment by andreyvelich** [comment]
It's great, thank you @RFMVasconcelos and @DomFleischmann !

> I'm happy to remove myself if you think we should stick to 2 people as @knkski and @DomFleischmann work with this code more on a daily basis.

It's fine to have more than 2 people in the approval list, so we can leave it as it is.

File: operators/OWNERS (hunk 1 above)
Comment (andreyvelich) [reply operators/OWNERS:4]: @RFMVasconcelos Answered here: https://github.com/kubeflow/katib/pull/1426#issuecomment-775230424.
It's fine to have more than 2 people.

Comment (RFMVasconcelos) [reply operators/OWNERS:4]: Awesome! :)

**Comment by andreyvelich** [comment]
@Bobgy @terrytangyuan Do you know why we still can't add @DomFleischmann to the OWNERS file ?
https://github.com/kubeflow/internal-acls/pull/431 PR was merged.

**Comment by Bobgy** [comment]
@DomFleischmann needs to accept an invitation from kubeflow org first.

**Comment by DomFleischmann** [comment]
> @DomFleischmann needs to accept an invitation from kubeflow org first.

Just joined, thank you!

**Comment by RFMVasconcelos** [comment]
Thank you @andreyvelich and @Bobgy !

**Pull request merged**